package.json

```diff
@@ -26,7 +29,17 @@
   "devDependencies": {
     "goodparts": "^1.1.0",
     "istanbul": "^0.4.5",
+    "nock": "^8.1.0",
     "pre-commit": "^1.1.3",
     "tape": "^4.6.2"
+  },
+  "dependencies": {
+    "env2": "^2.1.1",
+    "handlebars": "^4.0.5",
+    "hapi": "^15.1.1",
+    "hapi-auth-github": "^2.0.0",
```

Comment:
we _might_ want to add some error handling/view via: https://github.com/dwyl/hapi-error ...?